src/test/scala/is/hail/annotations/RegionValueToScala.scala

```diff
@@ -0,0 +1,51 @@
+package is.hail.annotations
+
+import is.hail.expr._
+
+import scala.reflect.ClassTag
+import scala.reflect.classTag
+
+object RegionValueToScala {
+  def classTagHail(t: Type): ClassTag[_] = t match {
+    case TBoolean(true) => classTag[Boolean]
+    case TBoolean(false) => classTag[java.lang.Boolean]
+    case TInt32(true) => classTag[Int]
+    case TInt32(false) => classTag[java.lang.Integer]
+    case TInt64(true) => classTag[Long]
+    case TInt64(false) => classTag[java.lang.Long]
+    case TFloat32(true) => classTag[Float]
+    case TFloat32(false) => classTag[java.lang.Float]
+    case TFloat64(true) => classTag[Double]
+    case TFloat64(false) => classTag[java.lang.Double]
+    case t => throw new RuntimeException(s"classTagHail does not handle $t")
+  }
+
+  def load[T: HailRep](region: Region, off: Long): T =
```

Comment:
These are only used in the tests, right?  Move them to the tests area.  Nobody in the compiler should be manipulating non-Code regions and offsets and I don't want to tempt fate.

Reply:
Oops, my mistake!